apps/prairielearn/src/components/EditQuestionPointsScore.html.ts

```diff
@@ -91,7 +91,11 @@ function EditQuestionPointsScoreForm({
       <input type="hidden" name="__action" value="edit_question_points" />
       <input type="hidden" name="__csrf_token" value="${csrfToken}" />
       <input type="hidden" name="instance_question_id" value="${instance_question.id}" />
-      <input type="hidden" name="modified_at" value="${instance_question.modified_at.toString()}" />
+      <input
+        type="hidden"
+        name="modified_at"
+        value="${instance_question.modified_at.toISOString()}"
```

Comment:
I'm not 100% convinced that this value would not be a string in some cases. Note that this component is sometimes used client-side, where the value runs through a JSON stringify/parse cycle.

https://github.com/PrairieLearn/PrairieLearn/blob/2a439af996b14eb347689d0f2e1be4fe67a28f5a/apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts#L80-L85
https://github.com/PrairieLearn/PrairieLearn/blob/2a439af996b14eb347689d0f2e1be4fe67a28f5a/apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts#L547-L557

Reply:
In Preact code we are already using SuperJSON to ensure that dates don't get implicitly converted to strings. I think we should also do that for client-side scripts.

Reply:
In this case the data is handled by bootstrap-table, I'd prefer not to rely on this conversion. I think it's safer to support a string version.

Reply:
This was done in #12420